base/intfuncs.jl

```diff
@@ -369,8 +369,18 @@ julia> powermod(5, 3, 19)
 ```
 """
 function powermod(x::Integer, p::Integer, m::T) where T<:Integer
-    p < 0 && return powermod(invmod(x, m), -p, m)
     p == 0 && return mod(one(m),m)
+    # When the concrete type of p is sigened and has the lowerst value,
```

Comment:
```suggestion
    # When the concrete type of p is signed and has the lowest value,
```

Reply:
This comment does not seem to be addressed, the typos are still there.

Reply:
Sorry for that…… I have corrected them